cockroachdb/values.yaml

```diff
@@ -374,6 +374,31 @@ tls:
     # Enable if the secret is a dedicated TLS.
     # TLS secrets are created by cert-mananger, for example.
     tlsSecret: false
+    # Enable if the you want cockroach db to create its own certificates
+    generator:
```

Comment:
I'd rather we name this API something like self-signer to clarify that this is cockroachdb signing its own certs

Reply:
Changed the name to `selfSigner`.